articles/openstack/ostack-sso-rcfile.md

```diff
@@ -0,0 +1,54 @@
+---
+title: How to use the OpenStack API using an SSO enabled user | UKCloud Ltd
+description: Explains the changes needed to make use of the OpenStack API with an SSO enabled user.
+services: openstack
+author: Steve Relf
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: OpenStack API with SSO users
+toc_fullpath: How To/ostack-sso-rcfile.md
+toc_mdlink: ostack-sso-rcfile.md
+---
+
+# How to use the OpenStack API using an SSO enabled user
+
+## Overview
+
+In order to use the OpenStack API with your SSO enabled user, you will need to make a few changes to your downloaded OpenStack RC file. This is to allow your API calls to be correctly passed to the SSO service.
+
+## Downloading your RC file.
+
+Please see [this guide](ostack-how-use-cli.md) section five for details on where to ger your RC file. You will require the v3 version. 
+
+## Modifying your RC file.
+
+Once you have downloaded your RC file you will need to add the following items to the bottom of the file.
+
+
+* export OS_IDENTITY_PROVIDER="sso"
```

Comment:
Would it make more sense to put these in a code block rather than as bullets?
```
export OS_IDENTITY_PROVIDER="sso"
export OS_PROTOCOL="oidc"
export OS_AUTH_TYPE="v3oidcpassword"
```